lib/display/display.ex

```diff
@@ -20,15 +20,17 @@ defmodule Inky.Display do
   def spec_for(type, accent \\ :black)
 
   def spec_for(type = :phat_ssd1608, accent) do
-    # Keep it minimal. Details are specified in `Inky.HAL.PhatSSD1608`.
     %__MODULE__{
       type: type,
       width: 250,
-      height: 122,
+      height: 136,
```

Comment:
Nice to see corrections :)!

Reply:
That magic number 136 was not explained in the Python library. The actual dimension is 250*122. Possibly it is because 122 is not divisible by 8.

So far my pHAT SSD1608 is working well. Before this PR, the image is slightly off center.

![inky-name-badge 20211109_073337](https://user-images.githubusercontent.com/7563926/140925687-b90d8f85-aeee-49ae-8ec9-c69b979c00e7.jpg)

Reply:
Top or bottom might be cropped. Yeah the Python library uses an image processing library to do some adjustment, but I am not familiar with all the image processing.

Note: In my fork, I make the display up side down so it is consistent with [manufacturer's example code](https://learn.pimoroni.com/article/getting-started-with-inky-phat).